HW1/SoftServeWOstl/FileWithStructur.cpp

```diff
@@ -0,0 +1,110 @@
+#include <cstdlib>
+#include <cstring>
+#include <fstream>
+#include <iostream>
+#include <string>
+
+namespace generator
+{
+    char BigSymvol[] = "АБВГҐДЕЄЖЗІКЛМНОПРСТУФХЦЧШЩЮЯ";
+    char SmalSymvol[] = "fбвгґдеєжзиіїйклмнопрстуфхцчшщьюя";
+ 
+int createFile(int argc, char** argv) 
+{
+    
+    setlocale(LC_ALL,".1251");
+
+    //для универсальності задамо питання кількості слів
+    int LenghFile = 0,i = 0,LenghName = 3;
+    std::string LastName;
+    
+    std::cout << "Скільки імен згенерувати?:\n";
+    std::cin>>LenghFile;    
+    std::srand ( time(NULL) );
+    
+    if (LenghFile < 1) return 0 ;
+    
+     // создаем файл (дублирую флаги по умолчанию)
+    std::ofstream NameFile("ListName.txt",std::ios_base::out | std::ios_base::trunc); 
+    
+    if (!NameFile.is_open())
+    {
+      std::cout << "Error open file!\n" ;
+      return 0 ;
+    }
+    
+    for (int count = 0;count < LenghFile;count++)
+    {
+        LenghName = 3+rand()%7; //Оскільки ми вирішили що потрібно випадкове ім'я, то воно має бути випадковим
+        LastName = "";
+        std::cout << "Довжина імені: " << LenghName << "\n";
+        for (i = 0;i < LenghName;i++)
+        {
+            //srand ( time(NULL) );
+            if (i==0) //Велика літера
+                {        
+                LastName.push_back(BigSymvol[(rand()%29 - 1)]);
+                }
+            else if (i<=LenghName) 
+                LastName.push_back(SmalSymvol[(rand()%33 - 1)]);
+         }
+        std::cout  << LastName << " \n";
+        //std::strncpy(WriteName, LastName, LenghName);
+        //WriteName[LenghName] = '\0';
```

Comment:
your commits must virtually never contain commented code. It's the first thing reviewer checks